src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -83,4 +118,9 @@ const StatNumber = styled('div')`
   }
 `;
 
+const StyledSectionValue = styled(SectionValue)`
+  margin: ${space(1)} 0;
+  color: #2c58a8;
```

Comment:
Should this use a theme color?

Reply:
Removed and turned into a Link which will have this by default.